pkg/controller/constrainttemplatestatus/constrainttemplatestatus_controller_test.go

```diff
@@ -306,6 +337,29 @@ violation[{"msg": "denied!"}] {
 		g.Eventually(verifyTStatusCount(ctx, c, 1), timeout).Should(gomega.BeNil())
 		g.Eventually(verifyCStatusCount(ctx, c, 1), timeout).Should(gomega.BeNil())
 	})
+
+	templateCpy = template.DeepCopy()
+	templateCpy.Spec.Targets[0].Rego = `
+	package foo
+
+	violtaion[invalid syntax error`
```

Comment:
this is still a sufficient test case without the typo right? can we remove the typo?

Reply:
@ritazh the typo was accidental, didn't notice it :D